1bash.template

```diff
@@ -591,10 +605,10 @@ BTCZ_PORT="5586"
 BTCZ_WORKER="$WORKERNAME"
 
 # Bitcoin Gold (BTG)
-BTG_ADDRESS="replace_with_your_address"
-BTG_EXTENSION_ARGUMENTS=""
-BTG_POOL="replace_with_your_BTG_POOL"
-BTG_PORT="replace_with_your_BTG_PORT"
+BTG_ADDRESS="$MPH_ADDRESS"
```

Comment:
should be "replace_with_your_address" to be consistent with all the other coins here which also have mph as pool example